update_key.py

```diff
@@ -18,11 +18,10 @@ def main(args):
     indent_spaces_count = 4
 
     # read .env
-    env = dotenv_values(".env")
-
+    env = dotenv_values(args.env_file)
     # Update initial ledger
 
-    ledger_filepath = "config/initial_ledger.json"
+    ledger_filepath = "base_config/initial_ledger.json"
```

Comment:
Add some argparse argument with default value for this

Reply:
Done